deepchem/models/torch_models/tests/test_progressive_multitask.py

```diff
@@ -14,30 +14,29 @@
 
 
 @pytest.mark.torch
-def test_construction():
-    """
-    Test that PrgressiveMultiTask Model can be constructed without crash.
+def test_progressivemultitask_construction():
+    """Test that PrgressiveMultiTask Model can be constructed without crash.
     """
 
     model = dc.models.torch_models.ProgressiveMultitask(
         n_tasks=1,
+        mode="classification",
         n_features=100,
         layer_sizes=[128, 256],
         alpha_init_stddevs=[0.08],
         weight_init_stddevs=0.02,
         bias_init_consts=1.0,
         activation_fns=nn.ReLU,
         dropouts=[0.2],
-        n_outputs=2,
+        n_classes=2,
     )
 
     assert model is not None
 
 
 @pytest.mark.torch
-def test_forward():
-    """
-    Test that the forward pass of ProgressiveMultiTask Model can be executed without crash
+def test_progressivemultitask_forward():
```

Comment:
We could add test for mode classification. Rest LGTM

Reply:
added it